packages/mdc-theme/README.md

```diff
@@ -79,130 +71,61 @@ $mdc-theme-background: #fff;
-| `mdc-theme--text-secondary-on-background` | Sets text to a suitable color for secondary text on top of the theme background. |
-| `mdc-theme--text-hint-on-background` | Sets text to a suitable color for hint text on top of the theme background. |
-| `mdc-theme--text-disabled-on-background` | Sets text to a suitable color for disabled text on top of the theme background. |
-| `mdc-theme--text-icon-on-background` | Sets text to a suitable color for icons on top of the theme background. |
-| `mdc-theme--text-primary-on-light` | Sets text to a suitable color for primary text on top of a light-colored background. |
-| `mdc-theme--text-secondary-on-light` | Sets text to a suitable color for secondary text on top of a light-colored background. |
-| `mdc-theme--text-hint-on-light` | Sets text to a suitable color for hint text on top of a light-colored background. |
-| `mdc-theme--text-disabled-on-light` | Sets text to a suitable color for disabled text on top of a light-colored background. |
-| `mdc-theme--text-icon-on-light` | Sets text to a suitable color for icons on top of a light-colored background. |
-| `mdc-theme--text-primary-on-dark` | Sets text to a suitable color for primary text on top of a dark-colored background. |
-| `mdc-theme--text-secondary-on-dark` | Sets text to a suitable color for secondary text on top of a dark-colored background. |
-| `mdc-theme--text-hint-on-dark` | Sets text to a suitable color for hint text on top of a dark-colored background. |
-| `mdc-theme--text-disabled-on-dark` | Sets text to a suitable color for disabled text on top of a dark-colored background. |
-| `mdc-theme--text-icon-on-dark` | Sets text to a suitable color for icons on top of a dark-colored background. |
+Some components can change their appearance when a theme-based modifier CSS class is applied. For example, `mdc-button--primary` will make the MDC Button the primary color. For more documentation on these modifier classes, consult the documentation for each component.
+
+If you want to modify an element, which is not a Material Design component, you can apply the following modifier CSS classes.
 
+CSS Class | Description
+--- | ---
+`mdc-theme--primary` | Sets the text color to the theme primary color
+`mdc-theme--accent` | Sets the text color to the theme accent color
+`mdc-theme--background` | Sets the background color to the theme background color
+`mdc-theme--primary-bg` | Sets the background color to the theme primary color
+`mdc-theme--accent-bg` | Sets the background color to the theme accent color
+`mdc-theme--text-<TEXT_STYLE>-on-<THEME_COLOR>` | Sets text to a suitable color for TEXT_STYLE on top of THEME_COLOR background
+`mdc-theme--text-<TEXT_STYLE>-on-light` | Sets text to a suitable color for TEXT_STYLE on top of light background
+`mdc-theme--text-<TEXT_STYLE>-on-dark` | Sets text to a suitable color for TEXT_STYLE on top of dark background
```

Comment:
I would duplicate the note above. People may be jumping down to this before they read about typography.